src/assets/styles/toolbar.scss

```diff
@@ -1,16 +1,75 @@
 .toolbar {
   text-align: center;
   background: $dark;
-  padding: 1rem;
```

Comment:
Oh, here the extra padding was making the div larger than 100% of the screen width.  I hear what you're saying though, about not using fixed dimensions/pixel values.